autoPyTorch/pipeline/components/setup/augmentation/image/Resize.py

```diff
@@ -35,7 +35,8 @@ def fit(self, X: Dict[str, Any], y: Any = None) -> BaseImageAugmenter:
 
     @staticmethod
     def get_hyperparameter_search_space(
-            dataset_properties: Optional[Dict[str, str]] = None
+            dataset_properties: Optional[Dict[str, str]] = None,
+            use_augmenter: Tuple[Tuple, bool] = ((True, False), True),
     ) -> ConfigurationSpace:
         cs = ConfigurationSpace()
         use_augmenter = CategoricalHyperparameter('use_augmenter', choices=[True, False], default_value=True)
```

Comment:
I think here you missed to use `use_augmenter`

Reply:
i sure did, thanks